#! /bin/sh /usr/share/dpatch/dpatch-run
## 0005_dbconverter.dpatch by  <fabbe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Build the dbconverter-2 utility and change the path to the sasldb
## DP: database file to /etc/sasldb2.

@DPATCH@
diff -urNad trunk~/utils/Makefile.am trunk/utils/Makefile.am
--- trunk~/utils/Makefile.am	2006-05-29 22:52:44.000000000 +0300
+++ trunk/utils/Makefile.am	2006-07-27 17:09:17.000000000 +0300
@@ -45,10 +45,10 @@
 all_sasl_libs = ../lib/libsasl2.la $(SASL_DB_LIB) $(LIB_SOCKET)
 all_sasl_static_libs = ../lib/.libs/libsasl2.a $(SASL_DB_LIB) $(LIB_SOCKET) $(GSSAPIBASE_LIBS) $(GSSAPI_LIBS) $(SASL_KRB_LIB) $(LIB_DES) $(PLAIN_LIBS) $(SRP_LIBS) $(LIB_MYSQL) $(LIB_PGSQL) $(LIB_SQLITE)
 
-sbin_PROGRAMS = @SASL_DB_UTILS@ @SMTPTEST_PROGRAM@ pluginviewer
+sbin_PROGRAMS = @SASL_DB_UTILS@ @SMTPTEST_PROGRAM@ pluginviewer dbconverter-2
 EXTRA_PROGRAMS = saslpasswd2 sasldblistusers2 testsuite testsuitestatic smtptest pluginviewer
 
-noinst_PROGRAMS = dbconverter-2
+#noinst_PROGRAMS = dbconverter-2
 
 if NO_SASL_DB_MANS
 man_MANS = 
diff -urNad trunk~/utils/dbconverter-2.c trunk/utils/dbconverter-2.c
--- trunk~/utils/dbconverter-2.c	2006-05-29 22:52:44.000000000 +0300
+++ trunk/utils/dbconverter-2.c	2006-07-27 17:09:54.000000000 +0300
@@ -385,7 +385,7 @@
 
 int main(int argc, char **argv)
 {
-    const char *db="/etc/sasldb";
+    const char *db="/etc/sasldb2";
     int result;
 
     if (argc > 1) {
@@ -421,7 +421,7 @@
     
     printf("\nThis program will take the sasldb file specified on the\n"
            "command line and convert it to a new sasldb file in the default\n"
-           "location (usually /etc/sasldb). It is STRONGLY RECOMMENDED that you\n"
+           "location (usually /etc/sasldb2). It is STRONGLY RECOMMENDED that you\n"
            "backup sasldb before allowing this program to run\n\n"
 	   "We are going to convert %s and our output will be in %s\n\n"
            "Press return to continue\n", db, db_new);
